From 1b83e396296f32f7fdd4cf430494e813e5573949 Mon Sep 17 00:00:00 2001
From: Daniel Leung <daniel.leung@intel.com>
Date: Tue, 14 May 2024 09:52:26 -0700
Subject: [PATCH 5/5] target/xtensa: fix sample_controller32 build for QEMU 7

The original patches for adding sample_controller32 are based on
QEMU 9. The GDB stub header file is not in the same place between
these two version. So fix that for QEMU 7. This can be reverted
when we move to QEMU 9.

Signed-off-by: Daniel Leung <daniel.leung@intel.com>
---
 target/xtensa/core-sample_controller32.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/target/xtensa/core-sample_controller32.c b/target/xtensa/core-sample_controller32.c
index 94e75ad6ef..1e6028bc17 100644
--- a/target/xtensa/core-sample_controller32.c
+++ b/target/xtensa/core-sample_controller32.c
@@ -1,6 +1,6 @@
 #include "qemu/osdep.h"
 #include "cpu.h"
-#include "gdbstub/helpers.h"
+#include "exec/gdbstub.h"
 #include "qemu/host-utils.h"
 
 #include "core-sample_controller32/core-isa.h"
-- 
2.43.0

